bundler/lib/bundler.rb

```diff
@@ -42,6 +42,7 @@ module Bundler
   autoload :Checksum,               File.expand_path("bundler/checksum", __dir__)
   autoload :CLI,                    File.expand_path("bundler/cli", __dir__)
   autoload :CIDetector,             File.expand_path("bundler/ci_detector", __dir__)
+  autoload :CompactIndexClient,     File.expand_path("bundler/compact_index_client", __dir__)
```

Comment:
Agree the `autoload` makes more sense here with the rest of the stuff.

Reply:
Yeah, it now fails without this change so I had to move it. However, I agree it makes more sense here.